module/VuFind/tests/integration-tests/src/VuFindTest/Mink/SearchFacetsTest.php

```diff
@@ -1146,4 +1244,110 @@ public function testOrFacets(): void
             $this->findCssAndGetText($page, '.search-header .search-stats')
         );
     }
+
+    /**
+     * Data provider for testCheckboxFacets
+     *
+     * @return array
+     */
+    public static function checkboxFacetselectionProvider(): array
+    {
+        $result = [];
+        foreach ([false, true] as $selectMulti) {
+            foreach ([false, true] as $unselectMulti) {
+                $params = '(' . ($selectMulti ? 'multi' : 'single') . '/' . ($unselectMulti ? 'multi' : 'single') . ')';
+                $result["select one $params"] = [
+                    ['Books'],
+                    8,
+                    $selectMulti,
+                    $unselectMulti,
+                ];
+                $name = "select two $params";
+                $result[$name] = [
+                    ['Books', 'Fiction'],
+                    7,
+                    $selectMulti,
+                    $unselectMulti,
+                ];
+            }
+        }
+        return $result;
+    }
+
+    /**
+     * Test checkbox facet selection
+     *
+     * @param array $checkFacets   Facet checkboxes to check
+     * @param int   $expectedCount Expected result count
+     * @param bool  $selectMulti   Select multiple?
+     * @param bool  $unselectMulti Unselect multiple?
+     *
+     * @dataProvider checkboxFacetselectionProvider
```

Comment:
...and again here, so I guess it's consistently inconsistent. ;-)

```suggestion
     * @dataProvider checkboxFacetSelectionProvider
```

Reply:
Fixed, now consistently consistent. :)